wdfn-server/waterdata/templates/macros/components.html

```diff
@@ -9,6 +9,38 @@
             </p>
         </div>
         <div class="select-time-series-container"></div>
+        <div class="graph-data usa-accordion">
+            <h2 class="usa-accordion__heading">
+                <button class="usa-accordion__button" aria-expanded="false" aria-controls="iv-download-container">
+                    Download this graph's data
+                </button>
+            </h2>
+            <div id="iv-download-container" class="usa-accordion__content usa-prose">
+                <ul class="usa-fieldset usa-list--unstyled">
```

Comment:
This is the part I would render in code rather than here.

Reply:
Do you mean the entire list? I did it this way for a couple reasons, the main one was that this keeps the list order intact. Even though the 'compare' and 'median' only exist when the user selects them, I wanted them to always appear in the same order no matter if the user selected median first and compare second. That would be difficult to control in JavaScript. The second reason, was I thought you mentioned in a previous review that we should do as much structure in the template and as little creation in JavaScript as possible.